console/src/app/components/feature-toggle/feature-toggle.component.ts

```diff
@@ -0,0 +1,54 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { CopyToClipboardModule } from '../../directives/copy-to-clipboard/copy-to-clipboard.module';
+import { CopyRowComponent } from '../copy-row/copy-row.component';
+import { InfoSectionModule } from 'src/app/modules/info-section/info-section.module';
+import { ToggleState, ToggleStateKeys, ToggleStates } from '../features/features.component';
+import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import { Source } from 'src/app/proto/generated/zitadel/feature/v2beta/feature_pb';
+import { FormsModule } from '@angular/forms';
+import { GetInstanceFeaturesResponse } from 'src/app/proto/generated/zitadel/feature/v2/instance_pb';
+import { FeatureFlag } from 'src/app/proto/generated/zitadel/feature/v2/feature_pb';
+
+@Component({
+  standalone: true,
+  selector: 'cnsl-feature-toggle',
+  templateUrl: './feature-toggle.component.html',
+  styleUrls: ['./feature-toggle.component.scss'],
+  imports: [
+    CommonModule,
+    FormsModule,
+    TranslateModule,
+    MatButtonModule,
+    InfoSectionModule,
+    MatTooltipModule,
+    CopyToClipboardModule,
+    CopyRowComponent,
+    MatButtonToggleModule,
+  ],
+})
+export class FeatureToggleComponent {
+  @Input() featureData: Partial<GetInstanceFeaturesResponse.AsObject> = {};
+  @Input() toggleStates: any = {}; // TODO: use ToggleStates, but get rid of the any warning of ngModel
+  @Input() toggleStateKey: string = '';
+  @Output() toggleChange = new EventEmitter<void>();
+
+  protected ToggleState: any = ToggleState;
```

Comment:
Why are you using any here? If you just use automatic type inference it should be fine otherwise you can use typeof ToggleState as an explicit type if you want that.

Reply:
makes sense..